projects/wave-core-new/src/lib/map/map-layer.component.ts

```diff
@@ -25,11 +37,12 @@ type LayerChanges = any; // TODO: use correct type
 // tslint:disable-next-line:directive-class-suffix
 export abstract class MapLayerComponent<OL extends OlLayer, OS extends OlSource, L extends Layer> {
 
-    /**
-     * A raster or vector layer
-     */
-    @Input() layer: L;
+    @Input() layerId: number;
+    @Input() isVisible: boolean;
+    @Input() workflow: UUID;
+    @Input() symbology: AbstractSymbology;
 
+    // TODO: just use the map service
```

Comment:
What is this TODO about?

Reply:
I was thinking why we wire it this way and not just use the map service, but I can remove the comment.